common-util/src/main/java/org/sunbird/telemetry/util/SunbirdTelemetryEventConsumer.java

```diff
@@ -38,23 +38,30 @@ public static SunbirdTelemetryEventConsumer getInstance() {
   public void consume(Request request) {
     ProjectLogger.log("SunbirdTelemetryEventConsumer:consume called.", LoggerEnum.INFO.name());
     if (request != null) {
+      ObjectMapper mapper = new ObjectMapper();
       try {
-        String telemetryReq = new Gson().toJson(getTelemetryRequest(request));
+        String telemetryReq = mapper.writeValueAsString(getTelemetryRequest(request));
         ProjectLogger.log(
             "SunbirdTelemetryEventConsumer:consume telemetry request:" + telemetryReq,
             LoggerEnum.DEBUG.name());
-        String response = HttpUtil.sendPostRequest(getTelemetryUrl(), telemetryReq, getHeaders());
-        ProjectLogger.log(
-            "SunbirdTelemetryEventConsumer:consume: Request process status = " + response,
-            LoggerEnum.INFO.name());
+        try {
+          String response = HttpUtil.sendPostRequest(getTelemetryUrl(), telemetryReq, getHeaders());
+          ProjectLogger.log(
+              "SunbirdTelemetryEventConsumer:consume: Request process status = " + response,
+              LoggerEnum.INFO.name());
+        } catch (Exception e) {
+          ProjectLogger.log(
+              "SunbirdTelemetryEventConsumer:consume: Generic exception occurred in sending telemetry request = "
+                  + e.getMessage(),
+              e);
+          ProjectLogger.log(
+              "SunbirdTelemetryEventConsumer:consume: Failure request = "
+                  + mapper.writeValueAsString(getTelemetryRequest(request)),
+              LoggerEnum.INFO.name());
+        }
       } catch (Exception e) {
         ProjectLogger.log(
-            "SunbirdTelemetryEventConsumer:consume: Generic exception occurred in sending telemetry request = "
-                + e.getMessage(),
-            e);
-        ProjectLogger.log(
-            "SunbirdTelemetryEventConsumer:consume: Failure request = "
-                + new Gson().toJson(getTelemetryRequest(request)),
+            "SunbirdTelemetryEventConsumer:consume: Failure converting to String  ",
```

Comment:
this should not be info logging, send "e" object as argument - it will automatically log as exception and even trace will be available. in main message append e.getMessage() which will give some idea about the failure reason.